packages/widgets/src/blocks/legacy-widget/edit/widget-type-selector.js

```diff
@@ -27,6 +27,8 @@ export default function WidgetTypeSelector( { selectedId, onSelect } ) {
 
 	return (
 		<SelectControl
+			__nextHasNoMarginBottom
+			className="wp-block-legacy-widget__select-widget"
```

Comment:
It looks to me like this bottom margin was unintentional to begin with, and should be removed. What do you think?

Reply:
I thought the margin was intentional because it looks more centered with the margin added. And when it's removed, it moves down and looks slightly off? Or maybe it just appears that way as a comparison, or I've looked at it too long and can't see clearly anymore. 😂 

I went down the rabbit hole before publishing the PR trying to remove this margin by using VStack, but if a description was added to Placeholder, it looks off. 

| Currently with margin  | With margin removed |
| ------------- | ------------- |
| ![Screen Shot 2022-12-15 at 6 01 05 PM](https://user-images.githubusercontent.com/35543432/208008284-fbedaff8-8495-4576-b4fa-513055b3a0c3.png)  | ![Screen Shot 2022-12-15 at 6 00 55 PM](https://user-images.githubusercontent.com/35543432/208008301-fa81e06f-dd1f-4475-b935-8a77d0291c66.png)  |

Reply:
The contents of the placeholder should be vertically centred. It doesn't really matter how we do that.

Reply:
Similar to [this one](https://github.com/WordPress/gutenberg/pull/46448#discussion_r1055297516), if the thing is already geometrically centered and we need to add complexity for optical adjustments, it needs to be meaningful and intentional.

<img width="688" alt="Screenshot of the Legacy Widget placeholder with flexbox outlines that show it is geometrically centered" src="https://user-images.githubusercontent.com/555336/209115930-458999ed-3dc9-4092-a7e3-259e2ea53b8e.png">

And even if we _do_ decide that it is worth doing an optical adjustment, adding bottom margin to an inner element for every possible instance is not sustainable. If anything, I'd rather see that tweaked via things that can be done universally, for example the paddings on `.components-placeholder` or the line-height on the `.components-placeholder__label`.